install.sh

```diff
@@ -52,6 +52,15 @@ function wait_and_approve_csr(){
    return 0
 }
 
+function set_gateway_node() {
+   test=$(kubectl get no -l "liqonet.liqo.io/gateway=true" 2> /dev/null | wc -l)
+   if [ $test == 0 ]; then
+      node=$(kubectl get no -o jsonpath="{.items[-1].metadata.name}")
+      kubectl label no $node liqonet.liqo.io/gateway=true > /dev/null
+      address=$(kubectl get no -l "liqonet.liqo.io/gateway=true" -o jsonpath="{.items[0].status.addresses[0].address}")
+   fi
```

Comment:
```suggestion
      kubectl label no $node liqonet.liqo.io/gateway=true > /dev/null
   fi
   address=$(kubectl get no -l "liqonet.liqo.io/gateway=true" -o jsonpath="{.items[0].status.addresses[0].address}")
```

Otherwise the address variable gets no value if the node was already labeled